dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java

```diff
@@ -187,24 +191,28 @@ private AuthorizationPolicy getAuthorizationPolicyFromMessage(String credentials
                     if (idx < (authDecoded.length() - 1)) {
                         password = authDecoded.substring(idx + 1);
                     }
-                }
+                } Liberty change: end */
+				String authInfo[] = authDecoded.split(":");
+                String username = (authInfo.length > 0) ? authInfo[0] : "";
+				String password = (authInfo.length > 1) ? authInfo[1] : "";
 
-                AuthorizationPolicy policy = sc.getUserPrincipal() == null
-                    ? new AuthorizationPolicy() : new PrincipalAuthorizationPolicy(sc);
+				// Liberty change:line below is removed
+				AuthorizationPolicy policy = new AuthorizationPolicy();	// Liberty change:line is added
                 policy.setUserName(username);
                 policy.setPassword(password);
-                policy.setAuthorizationType(authType);
+                //policy.setAuthorizationType(authType); // Liberty change:line is removed
                 return policy;
             } catch (Base64Exception ex) {
                 // Invalid authentication => treat as not authenticated or use the Principal
             }
         }
+/*		Liberty change: if block below is removed
         if (sc.getUserPrincipal() != null) {
             AuthorizationPolicy policy = new PrincipalAuthorizationPolicy(sc);
             policy.setAuthorization(credentials);
             policy.setAuthorizationType(authType);
             return policy;
-        }
+        }	Liberty change: end */
```

Comment:
If the if block was removed by you, you can remove it. If you were just adding the Liberty change headers to a change we had already made you can keep it.

Reply:
We are keeping it